src/components/map/components/action/layer-action/delete-action/index.js

```diff
@@ -39,18 +39,30 @@ export class VlMapDeleteAction extends VlMapLayerAction {
   }
 
   /**
-   * Zet de functie die wordt uitgevoerd na het uitvoeren van de verwijder actie.
-   * Voor elke geselecteerde feature wordt de reject callback gebruikt om de feature te verwijderen of de reject callback zodat de feature niet verwijderd wordt.
+   * Configure the function that will be called after processing an action.
+   * For each selected feature, the resolve callback will be used to delete the feature or the reject callback if the feature does not need to be deleted.
    *
-   * @param {Function} callback functie met volgende argumenten:
-   *                            - {[ol.Feature]} de te verwijderen features
-   *                            - {Function} resolve callback met ol.Feature als argument die verwijderd wordt
-   *                            - {Function} reject callback zonder argument waarbij de highlight verwijderd wordt
+   * @param {Function} callback function with the following arguments:
+   *                            - {[ol.Feature]} the features that are requested to be removed
+   *                            - {Function} resolve callback with the to be removed ol.Feature as argument
+   *                            - {Function} reject callback without argument, to remove the highlighted feature(s)
    */
   onDelete(callback) {
     this.__callback = callback;
   }
 
+  /**
+   * Specifies if the action is allowed to be performed on a feature and/or a layer. Returns true by default.
+   *
+   * @param {Object} feature Openlayers feature
+   * @param {Object} layer Openlayers layer
+   *
+   * @Return {boolean} true if the action is allowed to be performed, false if the action may not be performed for the supplied feature and/or layer
+   */
+  appliesTo(feature, layer) {
+    return true;
```

Comment:
Waarom parameters definiëren als je toch gewoon true returnt?

Reply:
Ik doe dat in dit geval zodat het duidelijk(er) is voor lezers dat je `appliesTo` functie de feature en layer meekrijgt wanneer hij wordt opgeroepen.